src/app/public/modules/checkbox/checkbox.component.ts

```diff
@@ -10,11 +10,14 @@ import {
 } from '@angular/core';
 
 import {
-  AbstractControl,
   ControlValueAccessor,
   NgControl
 } from '@angular/forms';
 
+import {
+  SkyFormsUtility
+} from '../shared/forms-utility';
+
 /**
```

Comment:
Are comments like this now exposed through Steve's docs tools?

Reply:
No, just the `@Input()` and `@Output` properties will be shown in the docs.